record/record.controller.js

```diff
@@ -194,17 +195,25 @@
         * @param {bool} isModalUpdate if update happens through modal pop up
         */
         function readUpdatedTable(refObj, dataModel, idx, isModalUpdate){
+            var errorData = {};
             if (isModalUpdate || completed[refObj.uri] || updated[refObj.location.schemaName + ":" + refObj.location.tableName]) {
                 delete updated[refObj.location.schemaName + ":" + refObj.location.tableName];
+
                 (function (i) {
                     refObj.read(dataModel.pageLimit).then(function (page) {
                         dataModel.page = page;
                         dataModel.rowValues = DataUtils.getRowValuesFromPage(page);
                     }, function (error) {
                         console.log(error);
+                        errorData.redirectUrl = $rootScope.reference.unfilteredReference.contextualize.compact.appLink;
+                        errorData.gotoTableDisplayname = $rootScope.reference.displayname.value;
+                        error.errorData = errorData;
```

Comment:
Same comment as in `record.app.js`

Lines 208-210 are unnecessary here. When this error is thrown on line 211, it is caught by the catch statement right below it where you set all of these values again.